Assets/Scripts/Models/Character.cs

```diff
@@ -376,7 +376,7 @@ public void WriteXml(XmlWriter writer)
         foreach (Need n in needs)
         {
             int storeAmount = (int)(n.Amount * 10);
-            needString = needString + n.needType + ";" + storeAmount.ToString() + ":";
+            needString = needString + n.Type + ";" + storeAmount.ToString() + ":";
```

Comment:
Type would work good for all the clases, and even ObjectType too. But the inconsistency in the names is not great, since you need to remember which one is used for each class, or just check the class.